anvil/src/shell.rs

```diff
@@ -725,7 +725,20 @@ pub fn init_shell<BackendData: Backend + 'static>(
                     surface,
                     seat,
                 } => {
-                    warn!(log_ref, "Unhandled grab");
+                    debug!(log_ref, "Popup grab");
```

Comment:
Should we not check for already active grabs? Or should a popup-grab override a resize or move grab?

Reply:
Initially i had a check for that, but there was a race. The grab is called as a result of an input event, so it is possible that the DefaultGrab in Pointer has still set a ClickGrab which would make a simple check for an active grab fail. Also a grab on a nested popup could already have an active popup grab. But maybe it is possible to use a serial as a condition. Something like if there is a active grab with either the provided serial or the serial stored in the popup grab from the last grab it is ok to override the grab.